mitosheet/mitosheet/code_chunks/step_performers/set_cell_value_code_chunk.py

```diff
@@ -32,18 +32,21 @@ def get_code(self) -> List[str]:
         sheet_index = self.get_param('sheet_index')
         column_id = self.get_param('column_id')
         row_index = self.get_param('row_index')
-        old_value = self.get_param('old_value')
         new_value = self.get_param('new_value')
         type_corrected_new_value = self.get_execution_data('type_corrected_new_value')
 
-        code: List[str] = []
+
+        column_header = self.post_state.column_ids.get_column_header_by_id(sheet_index, column_id)
+        transpiled_column_header = get_transpiled_code_for_object(column_header)
 
         # If nothings changed, we don't write any code
-        if old_value == new_value:
-            return code
+        if str(self.prev_state.dfs[sheet_index].at[row_index, column_header]) == new_value:
+            return []
 
-        column_header = self.post_state.column_ids.get_column_header_by_id(sheet_index, column_id)
-        transpiled_column_header = column_header_to_transpiled_code(column_header)
+        if new_value == '':
+            new_value = None
```

Comment:
Should this come before the previous check so that we can check None values and return early?

Reply:
We would need to have a special check if they are both `is none`